pubspec.lock

```diff
@@ -14,7 +14,7 @@ packages:
       name: args
```

Comment:
Looks good so far. As a side note, I looked further into whether or not we should be committing pubspec.lock

According to https://www.dartlang.org/guides/libraries/private-files we don't need to commit it. 
Don't worry about it here, but I'll make a separate commit to clear the git cache of it and add it to the .gitignore

Reply:
Great! Thank you for the information.

Have you read about .iml files? I think we do not realy need put them into version control, do we?

Reply:
Yeah, those should be left out as well. If any of us do anything fancy with intelliJ settings, we could mess up each other's project structure view or settings by committing these haha

Reply:
Yeah, thought that this could happen.